### PR TITLE
docs(turndown): use correct package names

### PR DESCRIPTION
**Pull request by maxpatiiuk**
- [x] Update outdated package names
- [x] Update outdated package link in pacakge.json
- [x] Update outdated link in PR template

I would suggest you also either deprecate the following packages, or at least add a notice that their name has changed and that new releases are only going to come out behind the new name:
- https://www.npmjs.com/package/joplin-turndown
- https://www.npmjs.com/package/joplin-turndown-plugin-gfm

### CODE REVIEW

**Comment by maxpatiiuk** [comment]
I have read the CLA Document and I hereby sign the CLA

File: packages/turndown/README.md [hunk 1]
```diff
@@ -229,4 +229,4 @@ The plugin API provides a convenient way for developers to apply multiple extens
 
 ## License
 
-turndown is copyright © 2017+ Dom Christie and released under the MIT license.
+turndown is copyright © 2017+ Dom Christie and released under the MIT license.
```
Comment (laurent22) [inline packages/turndown/README.md]: Remove whitespace change please

File: packages/turndown-plugin-gfm/README.md [hunk 2]
```diff
@@ -50,4 +50,4 @@ turndownService.use(tables)
 
 ## License
 
-turndown-plugin-gfm is copyright © 2017+ Dom Christie and released under the MIT license.
+turndown-plugin-gfm is copyright © 2017+ Dom Christie and released under the MIT license.
```
Comment (laurent22) [inline packages/turndown-plugin-gfm/README.md]: Remove whitespace change please

Comment (maxpatiiuk) [reply packages/turndown-plugin-gfm/README.md]: Fixed.

My IDE has a lot of issues with this repository - inconsistent line endings, trailing whitespace and etc - I had to disable all the checks to be able to edit the files, though apparently not all checks have gotten disabled.

**Pull request merged**